plugin/src/test/java/io/spine/tools/gradle/bootstrap/BootstrapPluginTest.java

```diff
@@ -38,11 +38,12 @@
     private Project project;
 
     @BeforeEach
-    void setUp(@TempDir Path projectDir) {
```

Comment:
We discussed with @armiol today the `TempDir` issue in relation to [another PR](https://github.com/SpineEventEngine/core-java/pull/1319/files/ca745c5ca14d9fa9c0bdecb630cf23da855fcbb3#r525058389). 

The conclusion is that we need to stick to the JUnit API because it performs the clean-up automatically for us. Please migrate to using `projectDir` as an annotated package-access field with the `File` type. It should do the job.